vms/event/models.py

```diff
@@ -21,7 +21,7 @@ class Event(models.Model):
     address = models.CharField(
         max_length=75,
         validators=[
-            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\')]+$', ),
+            RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\')|(,)]+$', ),
```

Comment:
Ideally, we should be checking for space after a comma otherwise, this will accept
`Ro,om 2,V,alley1` which makes no sense and adds ambiguity

Reply:
Thank you for the correction, will fix it :)

Reply:
@Monal5031  Sometimes there will be difficulty for normal users to understand what is wrong with- `Whitefield,Bangalore` which is missing a space after comma. So in my opinion it is not necessary to check for space after comma. Do tell me your opinion.